projects/igniteui-angular/src/lib/core/styles/components/carousel/_carousel-theme.scss

```diff
@@ -82,6 +86,10 @@
         $button-hover-arrow-color: text-contrast($button-hover-background);
     }
 
+    @if not($button-disabled-arrow-color) and $button-disabled-background {
+        $button-disabled-arrow-color: text-contrast($button-disabled-background);
```

Comment:
This will produce either white or black, which may not be an ideal indication of a disabled element. We could pass the produced value through the rgba color function to add a bit of transparency to the color, say 87%.